src/components/Tabs/Tabs.tsx

```diff
@@ -132,45 +110,63 @@ const getElementsFromData = (data, useHash) => {
 function Tabs(props: React.PropsWithChildren<TabsProps>) {
   const {
     children,
-    className = "",
-    data,
+    contentData,
+    defaultIndex = 0,
     id = generateUUID(),
-    isFitted = false,
-    useHash = false,
     onChange,
-    index = 0,
-    align = "start",
+    useHash = false,
   } = props;
-  const baseClass = "tabs";
-  // TODO: later when styling
-  // const variant = getVariant(tabsType);
-  // const styles = useStyleConfig("Tabs", { variant });
-  // line,enclosed, enclosed-colored, soft-rounded, solid-rounded, unstyled
+  const styles = useMultiStyleConfig("Tabs", {});
+  const { tabs, panels } = getElementsFromContentData(contentData, useHash);
 
-  if (children && data?.length) {
+  if (children && contentData?.length) {
     console.warn(
       "Only pass children or data in the `data` props but not both."
     );
   }
 
-  const { tabs, panels } = getElementsFromData(data, useHash);
-
   return (
     <ChakraTabs
       id={id}
-      align={align}
-      className={bem(baseClass, [], "", [className])}
-      isFitted={isFitted}
       // The following lazy loads each panel whenever it is needed.
       isLazy
       onChange={onChange}
-      defaultIndex={index}
+      defaultIndex={defaultIndex}
       variant="enclosed"
-      // __css={styles}
     >
-      {data ? (
+      {contentData ? (
         <>
-          <TabList>{tabs}</TabList>
+          <Box __css={styles.tablistWrapper}>
```

Comment:
The result:
<img width="313" alt="Screen Shot 2021-09-08 at 2 41 31 PM" src="https://user-images.githubusercontent.com/1280564/132570525-eff3a2c5-3d2e-4160-9845-e1f3ef7e50fc.png">

Reply:
The CSS should be adjusted so the tab labels do not wrap.